tests/sentry/search/events/test_builder.py

```diff
@@ -1485,7 +1491,7 @@ def test_run_query_with_multiple_groupby_orderby_distribution(self):
         self.assertCountEqual(
             result["meta"],
             [
-                {"name": "transaction", "type": "UInt64"},
+                {"name": "transaction", "type": self.expected_tag_value_type},
```

Comment:
I don't really know where this meta info is used, but i think we don't care if it changes, right?